Build failure when building master branch with latest wlroots 211eb9d6

wlroots commit `9b55737c` removed `wlr_matrix.h`, causing build error.

gcc (GCC) 14.2.1 20240910
```
[76/340] Compiling C object sway/sway.p/desktop_output.c.o
FAILED: sway/sway.p/desktop_output.c.o 
ccache cc -Isway/sway.p -Isway -I../sway-git/sway -Iinclude -I../sway-git/include -Iprotocols -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/json-c -I/usr/include/libevdev-1.0 -I/usr/include/pango-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/wlroots-0.19 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -DWLR_USE_UNSTABLE -D_POSIX_C_SOURCE=200809L -Wno-unused-parameter -Wno-unused-result -Wno-missing-braces -Wno-format-zero-length -Wundef -Wvla '-DSYSCONFDIR="//etc"' '-DSWAY_VERSION="1.10-dev-8acb0482d (" __DATE__ ", branch '"'"'master'"'"')"' -fmacro-prefix-map=../sway-git/= -march=native -O3 -pipe -fno-plt -fexceptions -Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -g -ffile-prefix-map=/tmp/makepkg/sway-git/src=/usr/src/debug/sway-git -flto=auto -fPIE -pthread -MD -MQ sway/sway.p/desktop_output.c.o -MF sway/sway.p/desktop_output.c.o.d -o sway/sway.p/desktop_output.c.o -c ../sway-git/sway/desktop/output.c
../sway-git/sway/desktop/output.c:13:10: fatal error: wlr/types/wlr_matrix.h: No such file or directory
   13 | #include <wlr/types/wlr_matrix.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[93/340] Compiling C object sway/sway.p/desktop_tearing.c.o
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in build().
    Aborting..
```

clang version 19.1.7
```
[338/340] Compiling C object sway/sway.p/desktop_output.c.o
FAILED: sway/sway.p/desktop_output.c.o 
clang -Isway/sway.p -Isway -I../sway-git/sway -Iinclude -I../sway-git/include -Iprotocols -I/usr/include/cairo -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/json-c -I/usr/include/libevdev-1.0 -I/usr/include/pango-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/wlroots-0.19 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=c11 -DWLR_USE_UNSTABLE -D_POSIX_C_SOURCE=200809L -Wno-unused-parameter -Wno-unused-result -Wno-missing-braces -Wno-format-zero-length -Wundef -Wvla '-DSYSCONFDIR="//etc"' '-DSWAY_VERSION="1.10-dev-8acb0482d (" __DATE__ ", branch '"'"'master'"'"')"' -fmacro-prefix-map=../sway-git/= -march=native -O3 -pipe -fno-plt -fexceptions -Wp,-D_FORTIFY_SOURCE=3 -Wformat -Werror=format-security -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -Wno-incompatible-pointer-types -g -ffile-prefix-map=/tmp/makepkg/sway-git/src=/usr/src/debug/sway-git -flto=thin -fPIE -pthread -MD -MQ sway/sway.p/desktop_output.c.o -MF sway/sway.p/desktop_output.c.o.d -o sway/sway.p/desktop_output.c.o -c ../sway-git/sway/desktop/output.c
../sway-git/sway/desktop/output.c:13:10: fatal error: 'wlr/types/wlr_matrix.h' file not found
   13 | #include <wlr/types/wlr_matrix.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[339/340] Linking target swaybar/swaybar
ninja: build stopped: subcommand failed.
==> ERROR: A failure occurred in build().
    Aborting...
```

Removing the include in `sway/desktop/output.c` fix building.

I don't think sway is using any of the function defined in `wlr_matrix.h`.

--- a/sway/desktop/output.c
+++ b/sway/desktop/output.c
@@ -10,7 +10,6 @@
 #include <wlr/types/wlr_buffer.h>
 #include <wlr/types/wlr_alpha_modifier_v1.h>
 #include <wlr/types/wlr_gamma_control_v1.h>
-#include <wlr/types/wlr_matrix.h>
 #include <wlr/types/wlr_output_layout.h>
 #include <wlr/types/wlr_output_management_v1.h>
 #include <wlr/types/wlr_output_power_management_v1.h>

